### DISCUSSION TITLE
I can't connect to github copilot

### DISCUSSION THREAD

**Discussion by ghost** [community/community#19726] [Copilot Conversations] [answered]
I've started the free trial but I can't connect to copilot even though I've used a personal access token with all the privileges. What do I do?
```
[INFO] [auth] [2022-06-29T22:38:46.612Z] Invalid copilot token: missing token: 404 
[ERROR] [default] [2022-06-29T22:38:46.613Z] GitHub Copilot could not connect to server. Extension activation failed: "User not authorized"
```

Is this option required? ` Allow GitHub to use my code snippets for product improvements *`

**Comment by davecheney** [comment, marked as answer]
@JediMaster25 which editor are you using?

**Reply by ghost** [reply]
vs code

**Reply by davecheney** [reply]
You shouldn't need to create a PAT, 
<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.

ps. Please double check you're not using vscodium by accident.

**Reply by ghost** [reply]
yeah, it's vs codium I thought they were the same. But I've followed the instructions for vs code in the [getting started guide](https://docs.github.com/en/copilot/getting-started-with-github-copilot/getting-started-with-github-copilot-in-visual-studio-code).

**Reply by davecheney** [reply]
Sadly the extension does not work with VSCodium at the moment. Please use vscode.

**Reply by ghost** [reply]
I'm using it on vs code and it seems to be working now.
